sky/skylet/providers/kubernetes/node_provider.py

```diff
@@ -257,9 +258,13 @@ def create_node(self, node_config, tags, count):
                     self.namespace, service_spec)
                 new_svcs.append(svc)
 
-        # Wait for all pods to be ready, and if it exceeds the timeout, raise an
-        # exception. If pod's container is ContainerCreating, then we can assume
-        # that resources have been allocated and we can exit.
+        # Wait for all pods including jump pod to be ready, and if it
+        # exceeds the timeout, raise an exception. If pod's container
+        # is ContainerCreating, then we can assume that resources have been
+        # allocated and we can exit.
+        jump_pod = kubernetes.core_api().read_namespaced_pod(
```

Comment:
Can we the SSH jump pod name from the `skypilot-ssh-jump` pod label instead of using `clouds.Kubernetes.SKY_SSH_JUMP_NAME`? That way in the future when jump pods are shared, this would still work.